src/components/maincontroller.js

```diff
@@ -1,930 +1,914 @@
-﻿define(['jellyfinactions', 'browserdeviceprofile'], function (jellyfinActions, deviceProfileBuilder) {
-    window.castReceiverContext = cast.framework.CastReceiverContext.getInstance();
-    window.mediaManager = window.castReceiverContext.getPlayerManager();
-    window.mediaManager.addEventListener(cast.framework.events.category.CORE,
-        event => {
-          console.log("Core event: " + event.type);
-          console.log(event);
-        });
-      
-    const playbackConfig = new cast.framework.PlaybackConfig();
-    // Set the player to start playback as soon as there are five seconds of
-    // media content buffered. Default is 10.
-    playbackConfig.autoResumeDuration = 5;
-
-    // According to cast docs this should be disabled when not needed
-    cast.framework.CastReceiverContext.getInstance().setLoggerLevel(cast.framework.LoggerLevel.DEBUG);
-
-    var init = function () {
+﻿import { factory as jellyfinActions } from "./jellyfinactions";
```

Comment:
I would standardize the filenames at some point.

Reply:
Yeah I agree